pkg/integrations/windows_exporter/config_windows.go

```diff
@@ -1,6 +1,9 @@
 package windows_exporter
 
-import "github.com/prometheus-community/windows_exporter/pkg/collector"
+import (
+	"github.com/prometheus-community/windows_exporter/pkg/collector"
+	col "github.com/prometheus-community/windows_exporter/pkg/collector"
```

Comment:
nit: importing the same package twice with a different alias; can we remove one?